presto-main/src/main/java/com/facebook/presto/operator/aggregation/noisyaggregation/NoisySumAvgAggregationUtils.java

```diff
@@ -22,13 +22,13 @@
 import static com.facebook.presto.common.type.DoubleType.DOUBLE;
 import static com.facebook.presto.spi.StandardErrorCode.INVALID_FUNCTION_ARGUMENT;
 
-public class NoisySumAggregationUtils
+public class NoisySumAvgAggregationUtils
 {
-    private NoisySumAggregationUtils()
+    private NoisySumAvgAggregationUtils()
     {
     }
 
-    public static void updateNoisySumState(NoisySumState state, double value, double noiseScale, Double lower, Double upper, Long randomSeed)
+    public static void updateNoisySumState(NoisySumAvgState state, double value, double noiseScale, Double lower, Double upper, Long randomSeed)
```

Comment:
Should the name of this function be updated? Could just be `updateState`, since the noisy/sum/avg stuff is implied by the parent class name.